### DISCUSSION TITLE
Cannot Export All Samples - SD Card Error

### DISCUSSION THREAD

**Discussion by dialectprg** [bangcorrupt/hacktribe#147] [Q&A] [answered]
Hi all,
thank you for the work youre doing with the E2... 

Sadly Id like to ask for a help... 

Ive hacked my grey E2 - i have all the sampler features available, and all the osc algorytms... 
Iam able to import samples with Oe2sSLE app. 

When I sample something and I try to export it to SD I get SD Card Error, and am not able to "keep" imported samples... 

***I also tryied reverting back to synth, but with no success.. 
- ive downloaded synth firmware, used the script e2-header.py with UNCOMMENTED lines 25+26... 
- here i get No File error (placed in KORG\Electribe sampler\system)

either - reverting back of fixing the sample saving issue is ok for me :)

thank you, Mike

**Comment by bangcorrupt** [comment, marked as answer]
In the latest version, hacktribe will look for files under the `hacktribe` directory.  Copy the `KORG/electribe sampler` directory to `KORG/hacktribe` and everything should work.

**Reply by dialectprg** [reply]
just wow... 
YOU JUST FIXED EVERYTHING.... EVERYTHING WORKS NOW...   MANY MANY THANKS....

greetings from Prague
